packages/fileeditor-extension/src/index.ts

```diff
@@ -32,63 +28,15 @@ import {
 
 import { ILauncher } from '@jupyterlab/launcher';
 
-import {
-  IEditMenu,
-  IFileMenu,
-  IMainMenu,
-  IRunMenu,
-  IViewMenu
-} from '@jupyterlab/mainmenu';
+import { IMainMenu } from '@jupyterlab/mainmenu';
 
 import { IStatusBar } from '@jupyterlab/statusbar';
 
-import { JSONObject, ReadonlyJSONObject } from '@phosphor/coreutils';
+import { JSONObject } from '@phosphor/coreutils';
 
 import { Menu } from '@phosphor/widgets';
 
-/**
- * The class name for the text editor icon from the default theme.
- */
-const EDITOR_ICON_CLASS = 'jp-MaterialIcon jp-TextEditorIcon';
-
-/**
- * The class name for the text editor icon from the default theme.
- */
-const MARKDOWN_ICON_CLASS = 'jp-MarkdownIcon';
-
-/**
- * The name of the factory that creates editor widgets.
- */
-const FACTORY = 'Editor';
-
-/**
- * The command IDs used by the fileeditor plugin.
- */
-namespace CommandIDs {
-  export const createNew = 'fileeditor:create-new';
-
-  export const createNewMarkdown = 'fileeditor:create-new-markdown-file';
-
-  export const changeFontSize = 'fileeditor:change-font-size';
-
-  export const lineNumbers = 'fileeditor:toggle-line-numbers';
-
-  export const lineWrap = 'fileeditor:toggle-line-wrap';
-
-  export const changeTabs = 'fileeditor:change-tabs';
-
-  export const matchBrackets = 'fileeditor:toggle-match-brackets';
-
-  export const autoClosingBrackets = 'fileeditor:toggle-autoclosing-brackets';
-
-  export const createConsole = 'fileeditor:create-console';
-
-  export const runCode = 'fileeditor:run-code';
-
-  export const runAllCode = 'fileeditor:run-all';
-
-  export const markdownPreview = 'fileeditor:markdown-preview';
-}
+import Commands, { EDITOR_ICON_CLASS, FACTORY } from './commands';
```

Comment:
I thought the purpose here was that other modules could use Commands. Can you also re-export Commands so that other modules can use it? Something like
```suggestion
export Commands from './commands';
import Commands, { EDITOR_ICON_CLASS, FACTORY } from './commands';
```

Reply:
Nice catch, I added the export, but it needed brackets since it was changed to a namespace.